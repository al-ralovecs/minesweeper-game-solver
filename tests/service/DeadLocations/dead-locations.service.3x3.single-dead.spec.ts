import { disposition } from '../../__fixtures__/state/3x3.single-dead.json';

import DeadLocationsService from '../../../src/service/dead-locations.service';
import getService from "./function.get.service";

describe('DeadLocationsService', () => {
    test('test if finds on a disposition', () => {
        expect(getService(disposition, 7).getDead).toMatchSnapshot();
    });
});
