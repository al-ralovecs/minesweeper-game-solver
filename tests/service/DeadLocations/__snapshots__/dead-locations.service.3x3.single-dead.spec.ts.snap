// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeadLocationsService test if finds on a disposition 1`] = `
AreaDto {
  "area": LocationSetDto {
    "data": Array [
      LocationDto {
        "x": 0,
        "y": 0,
      },
      LocationDto {
        "x": 1,
        "y": 0,
      },
      LocationDto {
        "x": 2,
        "y": 0,
      },
      LocationDto {
        "x": 0,
        "y": 1,
      },
      LocationDto {
        "x": 2,
        "y": 1,
      },
      LocationDto {
        "x": 0,
        "y": 2,
      },
      LocationDto {
        "x": 1,
        "y": 2,
      },
      LocationDto {
        "x": 2,
        "y": 2,
      },
    ],
  },
  "readOnlyArea": LocationSetDto {
    "data": Array [
      LocationDto {
        "x": 0,
        "y": 0,
      },
      LocationDto {
        "x": 1,
        "y": 0,
      },
      LocationDto {
        "x": 2,
        "y": 0,
      },
      LocationDto {
        "x": 0,
        "y": 1,
      },
      LocationDto {
        "x": 2,
        "y": 1,
      },
      LocationDto {
        "x": 0,
        "y": 2,
      },
      LocationDto {
        "x": 1,
        "y": 2,
      },
      LocationDto {
        "x": 2,
        "y": 2,
      },
    ],
  },
}
`;
